app/models/miq_expression/field.rb

```diff
@@ -2,7 +2,7 @@ class MiqExpression::Field
   FIELD_REGEX = /
 (?<model_name>([[:upper:]][[:alnum:]]*(::)?)+)
 \.?(?<associations>[a-z_\.]+)*
--(?<column>[a-z]+(_[a-z]+)*)
+-(?<column>[[a-z]]+(_[[:alnum:]]+)*)
```

Comment:
I'm wondering if `:alnum:` while convenient is too permissive? Does it make sense for it to have uppercase too?